src/detect.py

```diff
@@ -0,0 +1,23 @@
+import json
+import os
+
+
+def _get_card_definitions(dir_):
+    return {
+        i.replace(".json", ""): json.load(open(os.path.join(dir_, i)))
+        for i in next(os.walk(dir_))[2]
+        if i.endswith(".json")
+    }
+
+
+script_dir = os.path.dirname(os.path.realpath(__file__))
+
+CARD_DEFINITIONS = _get_card_definitions(os.path.join(script_dir, "card_definitions"))
+
+
+def get_card(record):
```

Comment:
maybe identify_card would be a clearer name?

Reply:
I agree, changed to `identify_card` and added a docstring